sdk/communication/communication-call-automation/src/callMedia.ts

```diff
@@ -71,14 +71,15 @@ export class CallMedia {
   public async play(
     playSources: FileSource[],
     playTo: CommunicationIdentifier[],
-    playOptions: PlayOptions = { loop: false }
+    playOptions: PlayOptions = { loop: false, operationContext: undefined }
```

Comment:
Do we need default value for operationContext? other options input doesn't have default value for operatonContext